### PR TITLE
Module 2 - Webpack config

### PR DESCRIPTION
**Pull request by TashaUA**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import React from 'react';
+import ReactDOM from "react-dom";
+
+import App from "./App";
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(
+<StrictMode>
+<App />
+</StrictMode>,
```
Comment (lotrien) [inline src/index.js]: It's a common practice to use .jsx format for files with React, you can benefit from it

File: webpack.common.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+module.exports = {
+        entry: {
+            app: './src/index.js',
+        },
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: "bundle.js"
+        },
+        resolve: {
+            extensions: ['.js', '.jsx']
```
Comment (lotrien) [inline webpack.common.js]: have a look on https://webpack.js.org/configuration/resolve/#resolvemodules
What are your thoughts about modules prop? Do you think we can use it here?

File: webpack.common.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        watch: false,
+        module: {
+            rules: [
+                {
+                    test: /\.(jsx|js)$/,
+                    exclude: /node_modules/,
+                    use: [{
+                        loader: 'babel-loader',
+                        options: {
+                            presets: [
```
Comment (lotrien) [inline webpack.common.js:27]: I just wondering why do we need to specify it here and in .bablerc as well?

Comment (TashaUA) [reply webpack.common.js:27]: As I understood, .bablerc is used to apply babel presets for some instruments that are not using webpack configs. I had an errors without it. But for now I checked with my final version and everything is working without it too.

File: src/index.js (hunk 1 above)
Comment (TashaUA) [reply src/index.js]: Fixed. Thanks in advance.

File: webpack.common.js (hunk 2 above)
Comment (TashaUA) [reply webpack.common.js]: Added resolve.modules section here....

**Comment by lotrien** [comment]
looks good to me 👍

**Pull request merged**